gemini/use-cases/retrieval-augmented-generation/code_rag.ipynb

```diff
@@ -0,0 +1,1231 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.    class CustomVertexAIEmbeddings(VertexAIEmbeddings):
</pre><p>Why do you need to create a new class instead of using the existing class from LangChain. If something needs to be updated in the library, you can make a PR for it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/481/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>